ios/ios-aptos-example/README.md

```diff
@@ -3,30 +3,64 @@
 [![Web3Auth](https://img.shields.io/badge/Web3Auth-SDK-blue)](https://web3auth.io/docs/sdk/pnp/ios)
 [![Web3Auth](https://img.shields.io/badge/Web3Auth-Community-cyan)](https://community.web3auth.io)
 
+
 [Join our Community Portal](https://community.web3auth.io/) to get support and stay up to date with the latest news and updates.
 
-This example demonstrates how to use Web3Auth to get create a Aptos account, connect and interact with Aptos chain in iOS.
+This example demonstrates how to use Web3Auth to create an Aptos account, connect to the Aptos chain, and interact with it in an iOS application using the [Web3Auth PnP Swift SDK](https://web3auth.io/docs/sdk/pnp/ios) and the [Aptos Swift SDK](https://github.com/ALCOVE-LAB/aptos-swift-sdk).
 
 ## How to Use
 
-### Download Manually
+### Installation
+
+1. **Download the Example Project**
+
+   ```bash
+   npx degit Web3Auth/web3auth-pnp-examples/ios/ios-aptos-example w3a-ios-aptos
+   ```
+
+2. **Install Dependencies**
+
+   Navigate to the project directory and install dependencies:
+
+   ```bash
+   cd w3a-ios-aptos
+   ```
```

Comment:
We can skip this step, it'll auto fetch deps when you open project in Xcode

Reply:
Done